benchmarks/src/main/java/io/druid/benchmark/FilterPartitionBenchmark.java

```diff
@@ -228,17 +226,15 @@ public void tearDown() throws IOException
 
   private IncrementalIndex makeIncIndex()
   {
-    return new OnheapIncrementalIndex.Builder()
+    return new IncrementalIndex.Builder()
         .setIncrementalIndexSchema(
             new IncrementalIndexSchema.Builder()
```

Comment:
This is a very frequent pattern, suggested to add shortcut like `IncrementalIndex.Builder. setSimpleIncrementalIndexSchema(AggregatorFactory...)`